lib/main.dart

```diff
@@ -23,7 +23,7 @@ class CLNApp extends AppView {
 
   @override
   Widget build(BuildContext context) {
-    var setting = provider.get<Setting>();
+    var setting = provider.get<Setting>(); // returns the instance of setting
```

Comment:
```suggestion
    // returns the instance of setting
    var setting = provider.get<Setting>();
```

usually I preferer this form